src/mavsdk/core/param_value.h

```diff
@@ -109,6 +118,13 @@ class ParamValue {
         double,
         std::string>
         _value{};
+
+    // Consti10: hacky, returns true if this parameter needs the extended parameters' protocol
+    // (which is the case when its value is represented by a string)
```

Comment:
I think that's fine.

Reply:
yeah, the hacky part is not there in param_value but in mavlink_parameter_receiver that must return different numbers for param_count on extended or non-extended requests.

Reply:
So we have to have two storage containers for it?